jsmpp/src/main/java/org/jsmpp/util/RelativeTimeFormatter.java

```diff
@@ -20,71 +20,87 @@
 import java.util.TimeZone;
 
 /**
- * Relative time formatter is {@link TimeFormatter} implementation referred to
- * SMPP Protocol Specification v3.4 point 7.1.1.
- * 
- * @author uudashr
+ * Relative time formatter is {@link TimeFormatter} implementation referred to SMPP Protocol Specification v3.4 point
```

Comment:
Please be careful to keep line length to 80 characters, it is hard to review the diff in the Github web page and some other tools when lines are longer